lib/KCard/index.vue

```diff
@@ -13,12 +13,13 @@
       <slot name="title"></slot>
     </template>
     <template #default>
-      <div class="thumbnail">
+      <div :class="isThumbnailPresent ? 'thumbnail' : null">
```

Comment:
Do we need this div if isThumbnailPresent is false? Given that in that situation, thsi div would be just simply a div without any styles I think we can safely add a v-if="isThumbnailPresent".

Reply:
This whole condition will need to be changed https://github.com/learningequality/kolibri-design-system/pull/752#pullrequestreview-2265059042